code/modules/mob/living/carbon/human/human_inventory.dm

```diff
@@ -444,6 +444,11 @@
 	var/obj/item/storage/equipped_item = get_item_by_slot(slot_item)
 	if(ismecha(loc) || HAS_TRAIT(src, TRAIT_HANDS_BLOCKED))
 		return
+	if(ismodcontrol(equipped_item)) // Check if the item is a MODsuit
+		if(thing && equipped_item.can_be_inserted(thing)) // Check if the item can be inserted into the MODsuit
+			equipped_item.handle_item_insertion(thing) // Insert the item into the MODsuit
+			playsound(loc, "rustle", 50, 1, -5)
+			return
```

Comment:
It seems you accidently have this on your branch, which means you probably accidently commited this to master. You'll want to remove this

Reply:
Haha yes i already saw as much, thank you however